### PR TITLE
docs(python): enable web-first assertions 

### PR DESCRIPTION
**Pull request by mxschmitt**

### CODE REVIEW

**Review by dgozman — APPROVED** [review]

File: docs/src/api/class-locatorassertions.md [hunk 1]
```diff
@@ -170,115 +290,220 @@ Note that if array is passed as an expected value, entire lists can be asserted:
...
+## method: LocatorAssertions.NotToBeVisible
+* langs: python
+
+Opposite of [`method: LocatorAssertions.toBeVisible`].
+
+### option: LocatorAssertions.NotToBeVisible.timeout = %%-assertions-timeout-%%
 
 ## method: LocatorAssertions.not
 - returns: <[LocatorAssertions]>
+* langs: java
```
Comment (yury-s) [inline docs/src/api/class-locatorassertions.md]: should go before `- returns`

File: packages/playwright-test/src/matchers/matchers.ts [hunk 2]
```diff
@@ -224,6 +224,18 @@ export function toHaveText(
   }
 }
 
+export function toHaveValue(
```
Comment (mxschmitt) [inline packages/playwright-test/src/matchers/matchers.ts:227]: drive-by: order change.
